README.md

```diff
@@ -1,71 +1,53 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/9ec787d03742d3d3fd8d/maintainability)](https://codeclimate.com/github/andela/ah-alpha-frontend/maintainability) <a href="https://codeclimate.com/github/andela/ah-alpha-frontend/test_coverage"><img src="https://api.codeclimate.com/v1/badges/9ec787d03742d3d3fd8d/test_coverage" /></a> <img src="https://img.shields.io/badge/reviewed%20by-Hound-blueviolet.svg" alt="hound"/> <img src="https://img.shields.io/badge/Team-Alpha-green.svg" alt="team"/>
 [![Build Status](https://travis-ci.org/andela/ah-alpha-frontend.svg?branch=develop)](https://travis-ci.org/andela/ah-alpha-frontend)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# AUTHORS HAVEN
 
-## Available Scripts
+Website: [Authors Haven](https://ah-alpha-frontend-staging.herokuapp.com/)
 
-In the project directory, you can run:
+Authors Haven is a web application where users can express out their thoughts through writing and can also read articles written and shares out by other authors.
 
-### `npm start`
+## SetUp
+This project has the following core dependencies:
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+    -Webpack
+    -Babel
+    -React
+    -Semantic-UI for styling
 
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+To SetUp the project clone the repo:
 
-### `npm test`
+1. `git clone https://github.com/andela/ah-alpha-frontend.git`.
+2. Run `npm install` to install all the dependencies.
+3. Add the following the following firebase config variables:
+    -apiKey: "AIzaSyCE9r0eKaMjQFbY2XaUMa_X4JvOODWdbVs",
+    -authDomain: "alpha-authors-haven-230108.firebaseapp.com"
+4. Run `npm start` to start the application.
 
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Functionality
+This application allows the user to perform the following activities:
 
-### `npm run build`
+## Authentication
+Authentication happens when:
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+1. A user can Signup and be verified to Login.
+2. A user can LogIn using their social networks.
 
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
+There are two users:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- Authenticated users
+- Non-authenticated users
 
-### `npm run eject`
+Authenticated users can access all funtionality of the application such as:
```

Comment:
You have not listed the functionality for creating, editing and deleting an article